lib/chromedriver.js

```diff
@@ -737,11 +737,14 @@ export class Chromedriver extends events.EventEmitter {
         : {desiredCapabilities: this.capabilities};
     this.log.info(
       `Starting ${this.desiredProtocol} Chromedriver session with capabilities: ` +
-        JSON.stringify(sessionCaps, null, 2)
+        JSON.stringify(sessionCaps, null, 2),
+    );
+    const {capabilities} = /** @type {NewSessionResponse} */ (
```

Comment:
sorry, looks like there were some auto lint changes. this is the main change to review here.